### ISSUE TITLE
Default `Deny` new Space Type creation in Group  Permissions 

### ISSUE THREAD

**Issue by luke-** [humhub/humhub-modules-enterprise-theme#25]
Instead of `Allow` as default new Space Types in the User -> Group -> Permission

**Comment by sunshinebard** [comment]
thinking it through till the end: to be able to set **user group defaults**
so I would have a group **profile** with no permissions, one with average permissens (e.g. to create spaces) and Groups with many permissions. That would be helpful within big networks

**Comment by luke-** [comment]
@sunshinebard I've added another feature request here: https://github.com/humhub/humhub/issues/6450

**Comment by luke-** [comment]
@Semir1212 Should we change this in the next major version?

**Comment by Semir1212** [comment]
Yes @luke- 

We should rework the default group permissions in general. New issue?

**Comment by luke-** [comment]
@Semir1212 Yes, a separate issue (maybe also per module) would be better. But we must be careful here....

**Comment by sunshinebard** [comment]
would that be attractive as part of the enterprise module?
